app/qml/layers/LayerDetail.qml

```diff
@@ -202,6 +202,33 @@ Page {
               }
             }
           }
+
+          Column {
+            id: layerAttribution
+
+            Layout.fillWidth: true
+            Layout.preferredHeight: attributionText.height + attributionTitle.height + 2 * InputStyle.panelMarginV2
+
+            spacing: InputStyle.panelSpacing
+
+            Text {
+              id: attributionTitle
+
+              text: qsTr( "Attribution" )
+              font.bold: true
+              font.pixelSize: InputStyle.fontPixelSizeNormal
+            }
+
+            Text {
+              id: attributionText
+
+              width: parent.width
+              textFormat: Text.RichText
+              wrapMode: Text.WordWrap
+              text: __inputUtils.layerAttribution(layerDetailData.mapLayer)
+            }
```

Comment:
It would be good to show the whole section only when the attribution text is not empty...